mimoto-default.properties

```diff
@@ -131,7 +131,7 @@ mosip.mandatory-languages=eng,khm
 
 #----------------------- CBEFF Util--------------------------------------------------
 # Cbeff URL where the files will be stored in git, change it accordingly in case of change of storage location.
-mosip.kernel.xsdstorage-uri=${spring.cloud.config.uri}/print/${spring.profiles.active}/${spring.cloud.config.label}/
+mosip.kernel.xsdstorage-uri=https://raw.githubusercontent.com/mosip/mosip-config/collab1/
```

Comment:
```suggestion
mosip.kernel.xsdstorage-uri=https://raw.githubusercontent.com/mosip/mosip-config/develop/
```

Reply:
No, we want to refer this based on spring profile